e2e/pages/enterRespondents/enterRespondents.js

```diff
@@ -4,70 +4,74 @@ const postcodeLookup = require('../../fragments/addressPostcodeLookup');
 module.exports = {
 
   state: {
-    context: 'firstRespondent',
+    context: 0,
   },
 
-  fields: function() {
+  fields: function () {
     const id = this.state.context;
 
     return {
       respondent: {
-        name: `#respondents_${id}_name`,
-        dob: {
-          day: `#respondents_${id}_dob-day`,
-          month: `#respondents_${id}_dob-month`,
-          year: `#respondents_${id}_dob-year`,
+        party: {
```

Comment:
I'd remove `party` element as it adds no value. Everything is inside anyway simpler structure such as 👇 is better IMO.

```
respondent: {
    partyType: `#respondents1_${id}_party_partyType`,
    title: `#respondents1_${id}_party_title`,
   ...
}
```

Reply:
agreed - adds uneccessary nesting